modules/shared_vpc/variables.tf

```diff
@@ -165,8 +165,8 @@ variable "disable_dependent_services" {
   type        = bool
 }
 
-variable "shared_vpc_enabled" {
-  description = "If shared VPC should be used"
+variable "enable_shared_vpc_service_project" {
```

Comment:
Please leave the name unchanged on outer modules for backwards-compatibility.

Reply:
@morgante you suggested renaming this variable. You meant we should have two different names in different places for the same variable? I'm confused now.